rotkehlchen/chain/ethereum/modules/optimism_gateway/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .accountant import OptimismGatewayAccountant  # noqa: F401
```

Comment:
rename the entire module, commit names, PR,  everywhere. It's not optimism gateway. It's optimism bridge. Gateway is just the website name, since it has docs, airdrop claim, governance etc.